apps/src/aichat/aichatApi.ts

```diff
@@ -206,7 +206,12 @@ async function postChatCompletionAsyncPolling(
   }
 
   return {
-    messages: getUpdatedMessages(newMessage, modelResponse, executionStatus),
+    messages: getUpdatedMessages(
```

Comment:
NIt: any reason your map to add requestId couldn't happen here (rather than passing into this function)?

Reply:
Only that I didn't think of it, haha. This would definitely simplify the change, and I wasn't super happy having to put `break;`s in. Thank you!